central_v2/src/components/GameCard.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { useEffect, useState } from 'react';
+import { APIClients, IGameTemplate, GameTemplate } from '@righton/networking';
+import { Box, Typography, Button } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import littleButton from '../images/littleButton.svg';
+import gym from '../images/gym.svg';
+import heart from '../images/heart.svg';
+import eyeball from '../images/eyeball.svg';
+import rocket from '../images/rocket.svg';
+
+interface StyledGameCardProps {
+  id: string;
+  title: string;
+  description: string;
+  image: string;
+  apiClients: APIClients;
+  game: IGameTemplate;
+}
+
+const LittleButtonSVG = styled('img')({
```

Comment:
so this component is a good example of one where we can substitute a lot of the hardcoded styling values to reference the theme. We should be able to clear out any of the hardcoded padding and margins, and hopefully some of the colours too